shared/templates/audit_rules_privileged_commands/oval.template

```diff
@@ -24,7 +24,7 @@
   <ind:textfilecontent54_object id="object_{{{ ID }}}_augenrules" version="1">
     <ind:filepath operation="pattern match">^/etc/audit/rules\.d/.*\.rules$</ind:filepath>
 {{% if product in ["sle12", "sle15"] %}}
-    <ind:pattern operation="pattern match">^[\s]*-a[\s]+always,exit[\s]+-F[\s]+path={{{ PATH }}}(?:[\s]+-F[\s]+perm=x)?[\s]+-F[\s]+auid>={{{ auid }}}[\s]+-F[\s]+auid!=(?:4294967295|unset)[\s]+(?:-k[\s]+|-F[\s]+key=)[\S]+[\s]*$</ind:pattern>
+    <ind:pattern operation="pattern match">^[\s]*-a[\s]+always,exit[\s]+(-S[\s]+all[\s]+)*-F[\s]+path={{{ PATH }}}(?:[\s]+-F[\s]+perm=x)?[\s]+-F[\s]+auid\>={{{ auid }}}[\s]+-F[\s]+auid!=(?:4294967295|unset|-1)[\s]+(?:-k[\s]+|-F[\s]+key=)[\S]+[\s]*$</ind:pattern>
```

Comment:
What is the purpose of this change?

Reply:
Well the DISA document definitions for SUSE, I've got, used as example bellow line:
```

-a always,exit -S all -F path=/sbin/unix_chkpwd -F perm=x -F auid>=1000 -F auid!=-1 -k privileged-unix-chkpwd
-a always,exit -S all -F path=/sbin/unix2_chkpwd -F perm=x -F auid>=1000 -F auid!=-1 -k privileged-unix2-chkpwd
```
So I decided to include also the possibility that the audit rule may include system calls as specification. It is is not mandatory part of the regex so should not interfere with the previous definition.

Reply:
I see. According to auditctl man page, using no -S or using -S all should be equivalent - Audit will watch for all syscalls in both cases. @stevegrubb  can you confirm this?
I see one important point which needs to be considered. By changing this template you are changing OVAL checks for many rules, 36 to be exact. I wonder if this should be somehow mirrored in their description. Because suddenly you will have passing rules where their Audit lines will not correspond with the rule description.
Thought son this? @ggbecker  @yuumasato

Reply:
Since this is a change that doesn't really affect the meaning of the audit line, I think the description doesn't need to be update, it would cause more confusion in my opinion. RHEL STIG doesn't mention the usage of this parameter for example.

Changing the subject, there is also a new backslash added to `auid\>={{{ auid }}}`. Is it really necessary? (The `-1` added to the `auid!=` should be ok though)

Reply:
The backslash seems to be an over-precaution in the regex, dropped it in 6c087d6

Reply:
> According to auditctl man page, using no -S or using -S all should be equivalent - Audit will watch for all syscalls in
> both cases. @stevegrubb can you confirm this?

The syscall is actually selected by the perm field for watches. You can't see any evidence of this when listing the rules because its done in a different place than the syscall filter. I think that in the absence of specifying a syscall, auditctl defaults to all. You do not need to specify it.

Reply:
Ok so if I understand correctly I can rely on 100% that `-S all` will not end up in the configuration file, so dropping that redundant group from the pattern